LBHackney-IT/lbh-fss-portal-frontend
# Feature/view organisation feedback

**description**

added functionality to allow VCSO users to view the status of their organisation with a corresponding reviewerMessage if is available 

**test**

- pull this branch 
- navigate to **/organisation** page - you should see a published organisation with an approved reviewer message (dummy text) displayed underneath it 
- switch to a user with a rejected organisation (do this by navigating to **mock-api/account/GET.js** and switching 'Mariano Luettgen' to 'Melody Zieme'
- then navigate back to **/organisation** page - you should see a rejected organisation with a rejected reviewer message (dummy text) displayed underneath it 

**ticket** 

https://app.clickup.com/t/9e5u88
